zephyr/Kconfig.memory

```diff
@@ -66,6 +66,11 @@ config LV_Z_DOUBLE_VDB
 	help
 	  Use two buffers to render and flush data in parallel
 
+config LV_Z_FULL_REFRESH
+	bool "Force full refresh mode"
+	help
+	  Force full refresh of display on update. Useful for displays where
+	  framebuffer is stored in RAM
```

Comment:
This is meaningless.  I do not know of any display controller where it is not stored in RAM. I guess what you mean is that it is performance gain to write complete area at once since display controller RAM is connected to CPU or is part of SoC's RAM.

Reply:
Fair point, this could be more clear. I updated the description to be the following:

```
Force full refresh of display on update. When combined with
LV_Z_VDB_SIZE, this setting can improve performance for display
controllers that require a framebuffer to be present in system memory,
since the controller can render the LVGL framebuffer directly
```